### ISSUE TITLE
DOC: Provide example on how to access functions that are globals of other functions

### ISSUE THREAD

**Issue by MImmesberger** [iza-institute-of-labor-economics/gettsim#587]
#### Location of the documentation

[This](https://gettsim.readthedocs.io/en/stable/tutorials/policy_functions.html#Changing-and-Replacing-Existing-Function(s)) part of the documentation.

#### Documentation problem

I tried to manipulate the `eink_st_ohne_kinderfreib_tu` policy function using alternative 1 described in the documentation, but couldn't find the function in the policy function dict. Apparently, intermediate policy functions can only be accessed as global variables of the main policy function, in my example: `policy_func_dict["eink_st_tu"].__globals__["eink_st_ohne_kinderfreib_tu"]`. I think this is not part of the documentation.

#### Suggested fix for documentation

Mention how to access functions that are global variables of the main policy function somewhere in the documentation.

**Comment by MImmesberger** [comment]
Pretty sure this became irrelevant after #805.
